### PR TITLE
add unit scale: bits/s, bytes/s, bytes

### PR DESCRIPTION
**Pull request by Wing924**
##### Summary
add unit scale: bits/s, bytes/s, bytes to UI

##### Component Name
web

### CODE REVIEW

File: web/gui/src/dashboard.js/units-conversion.js [hunk 1]
```diff
@@ -39,6 +53,14 @@ NETDATA.unitsConversion = {
...
             'TB/s': 1024 * 1024 * 1024
         },
+        'bytes': {
+            'B': 1,
+            'KB': 1024,
+            'MB': 1024 * 1024,
+            'GB': 1024 * 1024 * 1024,
+            'TB': 1024 * 1024 * 1024 * 1024,
+            'PB': 1024 * 1024 * 1024 * 1024 * 1024
+        },
```
Comment (ktsaou) [inline web/gui/src/dashboard.js/units-conversion.js]: Should these be `KiB`, `MiB`, `GiB`, etc?

File: web/gui/src/dashboard.js/units-conversion.js [hunk 2]
```diff
@@ -18,13 +18,27 @@ NETDATA.unitsConversion = {
...
             'gigabits/s': 1000000,
             'terabits/s': 1000000000
         },
+        'bytes/s': {
+            'bytes/s': 1,
+            'kilobytes/s': 1024,
+            'megabytes/s': 1024 * 1024,
+            'gigabytes/s': 1024 * 1024 * 1024,
+            'terabytes/s': 1024 * 1024 * 1024 * 1024
+        },
```
Comment (ktsaou) [inline web/gui/src/dashboard.js/units-conversion.js]: should these be `B/s`, `KiB/s`, `MiB/s`, `GiB/s`, etc?

**Comment by cakrit** [comment]
We should just modify #4912. "bytes" already exists, but it's called "B". 
"bytes/s" would similarly be called "B/s", this one is missing from #4912.

File: web/gui/src/dashboard.js/units-conversion.js (hunk 2 above)
Comment (gmosx) [reply web/gui/src/dashboard.js/units-conversion.js]: I thought we postponed the Kib/MiB etc units until the collectors are updated.

**Comment by gmosx** [comment]
@cakrit, I think we should close this PR, it's implemented in another PR?

**Comment by cakrit** [comment]
Already implemented in #4912

**Pull request closed without merging**